docs/1-introduction/1-3-models/1-3-1-texturing.md

```diff
@@ -0,0 +1,162 @@
+
+If FreeImage wasn't able to figure out the format, we'll return a `nullptr` because we won't be able to decode the image (if it was one in the first place), otherwise we'll start the decoding process with `FreeImage_LoadFromMemory(imageFormat, memHandle)` which will decode the image to a raw format consisting out of (usually) RGB(A) colors. 
+After that is done we can get rid of the raw memory as we no longer need it (we have the decoded data now).
+Something that isn't entirely required to do, but we will do for consistency is call `FreeImage_FlipVertical(image)`, D3D11's UV.y space are usually going up, where most other software expects it to go down, so naively loading images can result in textures being displayed upside down. 
+DirectXTex will handle this for us, so for consistency we'll flip the image as well.
+
+Now that we have the image in (almost) the format we need, we're getting to the texture creation. The sole exception being if the original texture was 24-bit (RGB), as there exist no 24-bit (R8G8B8) texture formats in D3D11 we'll need to convert it to a 32-bit (R8G8B8A8) texture before doing anything, this is easily done with `FreeImage_ConvertTo32Bits(image)`.
+
+Now we get to the D3D11 stuff.
+In order to create a texture we need a few things, its width, height and format. This will tell D3D11 what data we're giving it and how it should interpret it. 
+This data will go in a `D3D11_TEXTURE2D_DESC` which "DESC(ribes)" our texture data.
+
+Most images we see in our daily lives will be 24-bit or 32-bit, depending if they have alpha or not. As was noted before, D3D11 does not support 24-bit textures, but what do these values mean?
+The amount of "Bits per Pixel" is what we're talking about when we say an image is N-bit, this commonly translates to "how many channels" a texture has, in order to not overcomplicate things we'll assume all data we're working on is SDR (Standard Dynamic Range) and not HDR (High Dynamic Range)
+The key difference between those two is the range of a color, in SDR colors usually have 8 bits per channel, meaning our colors go from 0 to 255 (0xFF), HDR can take many shapes or forms and has no real consistent format to speak of but values can easily surpass the range of 0 to 1024 or more.
+
+Knowing that we assume SDR formats we'll `switch` on the amount of BPP of common values, anything weird and we're out!
+This is the first spot where we get into texture formats, our first case is "8 bits per pixel", not very common, but easy to handle, for us this means there only exists an "R" channel, no "GBA" to speak of. We'll tell D3D11 the same by specifying `DXGI_FORMAT_R8_UNORM`.
+
+You'll see the enum ends on `_UNORM`, this is 'how' our texture sampler will return the data to us in the shader.
+The following data types exist:
+
+- UNORM		: "Unsigned Normalized", data will be interpreted as a range of 0.0f to 1.0f.
```

Comment:
I believe there is some syntax for "table" structures, if possible turn this block from unorm-typeless into a table